Templates/ScriptCanvasNode/Template/README.md

```diff
@@ -0,0 +1,36 @@
+## How to use this template
+
+Use this template to produce a Script Canvas project that may hold Script Canvas nodes. 
+
+
+This guide will assume that you have created a gem called: `YourGem` using the command: 
+
+```
+scripts\o3de create-gem -gn YourGem -gp Gems\YourGem
```

Comment:
```suggestion
scripts\o3de create-gem -gn YourGem -gp <PATH\TO\YourGem>
```
We should make it clear their custom gem doesn't need to live inside their engine, they can create it anywhere on their machine.

Reply:
Good call, I will update